classes/PodsMeta.php

```diff
@@ -1202,6 +1202,11 @@ public function save_post( $post_id, $post, $update = null ) {
 		$pod  = self::$current_pod;
 		$data = array();
 
+		if ( !$pod ) {
```

Comment:
I appreciate this condition @jamesgol but we wanted to run some of the code below for non-Pods too.

What we want is to probably put the `$rest_enable` code and the subsequent return into a `if ( $pod ) {` block. I know this whole class is getting messy, we're going to have to clean things up at some point :/

Reply:
That is how the other functions added in the same commit do it (wrapping the rest code)